### ISSUE TITLE
V² > DIY > Intro > Intro

### ISSUE THREAD

**Issue by bcaswelch** [V-Squared/V-Squared.github.io#138]
# Change to Auto Index
# Sort in issues below
# Generic Tasks
## Create
- [x] Create issue to this article with _Path_ in issue name
- [ ] Copy article from → [Wiki Pages](https://github.com/V-Squared/v2-How-to-DIY-Mini-PC/wiki)
- [ ] Set _Readiness_ to _Create_
- [ ] Clean up Kramdown Code
  - [ ] Add lines between headings
  - [ ] Add lines in front of code blocks, such as tables
  - [ ] Change URL in tables to use _Referenced Links_
- [ ] Link Issue to Article
- [ ] Link Article to Issue in Reference
- [ ] Collect any writing Materials from other places to here
- [ ] Set _Readiness_ to _Develop_
## Develop
- [ ] **Table of Contents & Restructuring**
  - [ ] Add automatic _Table of Contents_
  - [ ] Restructure writing if need be
  - [ ] Add manual anchors and switch _Table of Contents_ to manual anchors
- [ ] **Update content**
  - [ ] If we reference to parts, find up to day parts
  - [ ] If ViCase or our philosophy changed, update it
- [ ] **Visual Placeholders**
  - [ ] Add text placeholder for graphics using _Code Block_ 
  - [ ] or find suitable placeholder graphics on the internet and edit to size: 853 x 480 px
  - [ ] Set Readiness to _Beta_
## Beta
- [ ] Trigger HC to _Beta Review_ by posting in comment: "# Request Beta Review"
- [ ] HC: Beta Review
- [ ] Include review feedback
- [ ] Finalize graphics as per feedback
## Make
- [ ] Trigger HC to _Make Review_ by posting in comment: "# Request Make Review"
- [ ] HC: Make Review
  - [ ] HC: Get feedback from at least one external editor
- [ ] Include review feedback
- [ ] Trigger HC to _Final Review_ by posting in comment: "# Request Final Review"
- [ ] HC: Make Final Review
- [ ] HC: Set _Readiness_ to _Make_
# Specific tasks to this Article
- Tell the story that V² DIY Workstations / ViPC compare great versus traditional Workstations.
- Use the best info-graphics from the comparison articles
- Basically Headline and Info-graphic must tell story for each of your points
- Find 3 ... 4 of the most powerful points to make 
# Reference
- [V²►Parts ►CPU](https://v-squared.github.io/diy/parts/cpu/)

**Comment by bcaswelch** [comment]
Image for ViCase Generations
![vicase-overview-1](https://cloud.githubusercontent.com/assets/16950279/16326887/88e78456-397f-11e6-9221-aa67d7b3a454.jpg)

**Comment by VillageHubertChen** [comment]
@bcaswelch 

Thanks. Good idea. It is not a perfect match but better than nothing. This image did not had enough resolution. I found the latest counter part. Sadly no coke bottle. We will make all new Title Images 3:1 aspect ratio. Since our default size is 480p = 853 x 480, the new size is: 853 x 284. Weird size. Here goes:

![vicase-three-sizes-title](https://cloud.githubusercontent.com/assets/7278716/16330471/06aeec16-3a1c-11e6-93f4-9fe13d719095.jpg)

I uploaded it to the Article

**Comment by bcaswelch** [comment]
@VillageHubertChen 

Looks good. Also added in the ViPC features into the article and they are ready for review. Will upload the new graphs soon.

**Comment by bcaswelch** [comment]
![workstationprice2](https://cloud.githubusercontent.com/assets/16950279/16353354/df3658ba-3a2d-11e6-8258-9bab7f532c21.png)

![workstationsize2](https://cloud.githubusercontent.com/assets/16950279/16353355/df3819ac-3a2d-11e6-948e-1cf4074840d2.png)
